.cirrus.yml

```diff
@@ -46,7 +43,6 @@ build_task:
 bindist_task:
   name: bindist
   depends_on:
-    - build-ghc-8.10.7
     - build-ghc-9.0.2
     - build-ghc-9.2.5
     - build-ghc-9.2.7
```

Comment:
suspicious? should just be 9.2.8?

Reply:
In theory, yes, but since we don't have any experience with freebsd, I, personally, don't know which ghc versions have a custom build available.

Additionally, Cirrus CI seems to discontinue their free freebsd CI, which will likely cause us to drop it in the next release. https://cirrus-ci.org/blog/2023/07/17/limiting-free-usage-of-cirrus-ci/